data_steward/analytics/cdr_ops/README.md

```diff
@@ -4,4 +4,130 @@ Notebooks for operational evaluation of data.
+1. Select the `View -> Cell Toolbar -> Tags` menu option to activate the notebook's tag view.
+2. In a single cell, define 1 or more variables that will be your parameters.  
+    1. A optional comment on a variable's line will serve as its description.
+    1. An optional type hint can be provided. 
+3. In your parameter-containing cell, enter the term **parameters** into the textbox adjacent to the <em>Add tag</em> button.
+4. Click the <em>Add tag</em> button to mark the cell with the entered tag.
+
+
+![Parameterization instructions](./images/parameter_definition.png)
+
+### Viewing Notebook Parameters
+
+To view the parameters required for a particular notebook, the <em>--help_notebook</em> option can be passed to `report_runner.py`.
+
+```
+$ python report_runner.py "Example Notebook.py" --help_notebook
+Parameters inferred for notebook Example Notebook:
+2021-01-28 17:44:08 - INFO - papermill - Input Notebook:  Example Notebook.ipynb
+  my_unknown_parameter: Unknown type (default "", required=True)
+                                  This parameter has an unknown type.
+  my_string_parameter: str (default "", required=True)
+                                  This is a string parameter.
+  my_int_parameter: int (default 0, required=False)
+                                  This is an integer parameter.
+```
+
+### Running Notebook
+
+When executing a notebook, each parameter is passed with a separate <em>-p</em> or <em>--parameters</em> flag.
+
+```
+$ python report_runner.py "Example Notebook.py"  
+        --output_path executed_notebook.html  
+         -p my_unknown_parameter "hello"  
+         -p my_string_parameter "world"  
+         -p my_int_parameter 6
+Executing: 100%|█████████████████████████████████████████████████████| 3/3 [00:01<00:00,  2.11cell/s]
+```
+And the outputted html:
+![Executed HTML](./images/html_image.png)
+
+Notice that an extra cell containing the passed parameter values was injected into the notebook.
+
+### Debugging
+
+There are a few sets of errors one may encounter when attempting to execute a notebook.
+
+#### Missing/Unknown Parameters
+
+These errors occur when required parameters or values are not passed in or an unknown parameter is provided. A error message is logged and the notebook is not executed:
+
+```
+$ python report_runner.py "Example Notebook.py" 
+    --output_path executed_notebook.html 
+     -p my_unknown_parameter "hello"  
+     -p my_int_parameter 6
+Missing the parameter my_string_parameter for notebook Example Notebook
+Parameters inferred for notebook Example Notebook:
+  my_unknown_parameter: Unknown type (default "", required=True)
+                                  This parameter has an unknown type.
+  my_string_parameter: str (default "", required=True)
+                                  This is a string parameter.
+  my_int_parameter: int (default 0, required=False)
+                                  This is an integer parameter.
+```
+
```

Comment:
Verified this by running the following command:

`python report_runner.py "rdr_export_qc.py" -p old_rdr rdr20201201 new_rdr rdr20210104`

This failed successfully because no project_id was supplied!